src/java/org/apache/cassandra/cql3/statements/schema/AlterKeyspaceStatement.java

```diff
@@ -62,6 +63,9 @@ public AlterKeyspaceStatement(String keyspaceName, KeyspaceAttributes attrs)
 
     public Keyspaces apply(Keyspaces schema)
     {
+        if (ClientWarn.instance.get() == null)
+            ClientWarn.instance.captureWarnings();
+        int previousWarningsSize = ClientWarn.instance.getWarnings() == null ? 0 : ClientWarn.instance.getWarnings().size();
```

Comment:
maybe we can introduce `ClientWarn::numWarnings()` method that would hide this null check and return 0 or size?

Reply:
Yeah on the other hand you can argue you're loosing the semantics on a non-inited `ClientWarn` instance returning `0` instead of NPE-ing... :shrug: I'd say this is a nit? I am weary of touching such a class for such a specific usage and a nit :shrug:

Reply:
up to you

Reply:
logic would not be lost:
```
int previousWarningsSize = ClientWarn.instance.numWarnings();
```